### DISCUSSION TITLE
Moveable Gird

### DISCUSSION THREAD

**Discussion by matt1208** [chakra-ui/chakra-ui#3829] [Q&A]
I am using SimpleGrid and Box to create a card style gird. Is there the ability with charka to allow users to move their cards (aka the boxes) around and customize where they have them on their screen?

**Comment by chr-ge** [comment]
If you're talking about dragging and dropping the cards around, you'll need a react package like: [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd).

**Reply by matt1208** [reply]
Will I be able to use this package with the SimpleGrid Chakra component? @chr-ge

**Reply by chr-ge** [reply]
yes

**Reply by matt1208** [reply]
Do you know of any examples/codesandboxes I can view of them both being used together? Sorry I am still confused on how this would integrate together. Thanks! @chr-ge

**Reply by chr-ge** [reply]
Here: https://codesandbox.io/s/goofy-wildflower-25n2l?file=/src/index.js

All I did was copy the [example from react-beautiful-dnd](https://codesandbox.io/s/zqwz5n5p9x?file=/src/index.js) and I replaced all the `div`s with chakra `Box`s.

**Reply by matt1208** [reply]
Sorry I didn't see the last reply! Would this also work if the boxes were in a SimpleGrid like 3x3?
